gms/gossiper.cc

```diff
@@ -2328,12 +2328,12 @@ bool gossiper::is_alive(inet_address ep) const {
     return is_alive;
 }
 
-future<> gossiper::wait_alive(std::vector<gms::inet_address> nodes, std::chrono::milliseconds timeout) {
+future<> gossiper::wait_alive(std::vector<gms::inet_address> nodes, std::chrono::milliseconds timeout) const {
```

Comment:
This one is just for cleanup. Since both functions do not modify the gossiper state they can be marked as const.
In general, I'd ask why NOT mark it as const.
There needs to be a reason to change the object to do so.
it shouldn't be the default as it makes it harder to implement methods that need to be const as they can't just call other mutable methods.

Reply:
I'll defer the cleanups to a followup PR